src/loc/pt-br.ts

```diff
@@ -0,0 +1,358 @@
+    ThemeColorsGroupName: "Cores do tema",
+    HighlightColorsGroupName: "Destacar cores",
+    StandardColorsGroupName: "Cores padrão",
+    CustomColorsGroupName: "Cores personalizadas",
+    ThemeColorDarker: "Tema mais escuro",
+    ThemeColorDark: "Tema escuro",
+    ThemeColorDarkAlt: "Tema escuro alternativo",
+    ThemeColorPrimary: "Tema principal",
+    ThemeColorSecondary: "Tema secundário",
+    ThemeColorTertiary: "Terciário neutro",
+    ThemeColorNeutralSecondary: "Neutro secundário",
+    ThemeColorNeutralPrimaryAlt: "Alternativo primário neutro",
+    ThemeColorNeutralPrimary: "Neutro primário",
+    ThemeColorNeutralDark: "Neutro escuro",
+    HighlightColorYellow: "Amarelo",
+    HighlightColorGreen: "Verde",
+    HighlightColorAqua: "Aqua",
+    HighlightColorMagenta: "Magenta",
+    HighlightColorBlue: "Azul",
+    HighlightColorRed: "Vermelho",
+    HighlightColorDarkblue: "Azul escuro",
+    HighlightColorTeal: "Cerceta",
+    HighlightColorDarkgreen: "Verde escuro",
+    HighlightColorPurple: "Roxo",
+    HighlightColorMaroon: "Marrom",
+    HighlightColorGold: "Ouro",
+    HighlightColorDarkgrey: "Cinza escuro",
+    HighlightColorGrey: "Cinza",
+    HighlightColorBlack: "Preto",
+    StandardColorDarkred: "Vermelho escuro",
+    StandardColorRed: "Vermelho",
+    StandardColorOrange: "Laranja",
+    StandardColorYellow: "Amarelo",
+    StandardColorLightgreen: "Luz verde",
+    StandardColorGreen: "Verde",
+    StandardColorLightblue: "Azul claro",
+    StandardColorBlue: "Azul",
+    StandardColorDarkblue: "Azul escuro",
+    StandardColorPurple: "Roxo",
+
+    DatePickerMonthLongJanuary: "Janeiro",
+    DatePickerMonthShortJanuary: "Jan",
+    DatePickerMonthLongFebruary: "Fevereiro",
+    DatePickerMonthShortFebruary: "Fevereiro",
+    DatePickerMonthLongMarch: "Marcha",
+    DatePickerMonthShortMarch: "Março",
+    DatePickerMonthLongApril: "Abril",
+    DatePickerMonthShortApril: "Abril",
+    DatePickerMonthLongMay: "Maio",
+    DatePickerMonthShortMay: "Maio",
+    DatePickerMonthLongJune: "Junho",
+    DatePickerMonthShortJune: "Junho",
+    DatePickerMonthLongJuly: "Julho",
+    DatePickerMonthShortJuly: "Julho",
+    DatePickerMonthLongAugust: "Agosto",
+    DatePickerMonthShortAugust: "Agosto",
+    DatePickerMonthLongSeptember: "Setembro",
+    DatePickerMonthShortSeptember: "Setembro",
+    DatePickerMonthLongOctober: "Outubro",
+    DatePickerMonthShortOctober: "Outubro",
+    DatePickerMonthLongNovember: "Novembro",
+    DatePickerMonthShortNovember: "Novembro",
+    DatePickerMonthLongDecember: "Dezembro",
+    DatePickerMonthShortDecember: "Dezembro",
+    DatePickerDayLongSunday: "Domigo",
+    DatePickerDayShortSunday: "Sol",
+    DatePickerDayLongMonday: "Segunda-feira",
+    DatePickerDayShortMonday: "Seg",
+    DatePickerDayLongTuesday: "Terça-feira",
+    DatePickerDayShortTuesday: "Ter",
+    DatePickerDayLongWednesday: "Quarta-feira",
+    DatePickerDayShortWednesday: "Casar",
```

Comment:
@joelfmrodrigues
I don't think this is correct. DatePickerDayShortSaturday: "Casar" -> means married... should be "Qua" i think...